pkgs/ok_http/android/build.gradle

```diff
@@ -23,12 +26,21 @@ rootProject.allprojects {
 }
 
 apply plugin: "com.android.library"
+apply plugin: 'kotlin-android'
 
 android {
     if (project.android.hasProperty("namespace")) {
         namespace = "com.example.ok_http"
     }
 
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
```

Comment:
Do you need this when you don't have any sources?

Reply:
I added them just as a pre-check to include all sources (next PR related to `testRedirects` will have Kotlin sources)

Should I remove them for now, and include in the next one?

Reply:
Yes, please add this to the PR that adds Kotlin code.

Reply:
Removed ✅